cluster-scope/base/core/namespaces/prometheus-ai-project/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/project-admin-rolebindings/prometheus-ai
+    - ../../../../components/limitranges/default
```

Comment:
please also add quota: small, medium, large

Reply:
This was not mentioned in our guides, can you please specify what do you mean?
Do you want us to add key 'quota' value 'medium' for this yaml file? or maybe something else.
If there is any example that would be great